### PR TITLE
improve README.md

### PR DESCRIPTION
**Pull request by caicancai**
### Task Description
The Chinese and English versions of readme are not completely aligned 

### Solution Description
Alignment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ English | [中文版](README_CN.md)
 
 - **Continuous Availability**
     
-    OceanBase Database adopts Paxos Consensus algorithm to achieve Zero RPO and less than 8 seconds of RTO.
+    OceanBase Database adopts Paxos Consensus algorithm to achieve Zero RPO and less than 8 seconds of RTO.Supports intra-city/remote disaster recovery, enabling multi-activity in multiple locations and zero data loss
```
Comment (hnwyllmm) [inline README.md:57]: Need a space(' ') before the word 'Supports' and a dot('.') at the end of the sentence.

**Review by hnwyllmm — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (caicancai) [reply README.md]: thank you

**Comment by caicancai** [comment]
@hnwyllmm

**Review by hnwyllmm — APPROVED** [review]
LGTM

**Comment by hnwyllmm** [comment]
Please sign the CLA.

**Comment by caicancai** [comment]
hello,I'm logged into CLA
屏幕截图 2023-09-04 151250

**Comment by hnwyllmm** [comment]
Can't you see the signature form?

**Comment by caicancai** [comment]
屏幕截图 2023-09-04 154632

This form, but I can't fill in the information

**Comment by hnwyllmm** [comment]
```
caicancai seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/oceanbase/oceanbase?pullRequest=1551) it.
```

It seems you committed code with wrong author information.
Please check the git log and make sure it shows your github email address.
If it doesn't show the right information, you can use `git amend --author` command to modify the author (the command maybe invalid and please check it on the website).

**Pull request closed without merging**